stop_svc.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+# CloudForms Retire Services - Ahsen Shah <ahshah@redhat.com>
+# this script is for automation only, be careful!
+# because this script reads stdin, you have to set those environment variables:
+# - username
+# - password
+# - uri
+
+# safety options
+set -e -o pipefail
+
+ORIG="$(cd "$(dirname "$0")" || exit; pwd)"
+
+# shellcheck source=common.sh
+. "${ORIG}/common.sh"
+
+
+usage() {
+    echo "Error: Usage $0 [-u <username> -w <uri> ]"
+}
+
+
+while getopts y:hu:w:d: FLAG; do
+    case $FLAG in
+#        y) noni=1;;
+        u) username="$OPTARG"
+           export username
+           ;;
+        w) uri="$OPTARG"
+           export uri
+           ;;
+        h) usage;exit;;
+        *) usage;exit;;
+    esac
+done
+
+
+if ! command -v jq > /dev/null; then
+    echo >&2 "please install jq"
+    exit 2
+fi
+
+
+while read -r itemID name; do
+    # ensure (itemID name) couple makes sense
+    remoteName=$(cfget "${uri}/api/services/${itemID}?attributes=name&expand=resources" \
+                     | jq -r '.name')
+
+    if [ "${name}" != "${remoteName}" ]; then
+        echo >&2 "name provided with id does not match remote name: ${name} != ${remoteName}"
+        exit 2
+    fi
+
+    PAYLOAD="{ \"action\": \"StopNV\" }"
```

Comment:
the 2 scripts, start and stop_svc.sh look very much the same. You could probably parameterize the action (ex: `-a start`)

```
lifecycle_svc.sh  ... -a start
lifecycle_svc.sh  ... -a stop
```